modules/__tests__/Media-test.js

```diff
@@ -24,52 +26,85 @@ describe('A <Media>', () => {
     node = document.createElement('div')
   })
 
-  describe('with a query that matches', () => {
+  describe('with a children function', () => {
+    beforeEach(() => {
+      window.matchMedia = createMockMediaMatcher(true)
+    })
+
+    it('should render expected result of the children function call', () => {
+      beforeEach(() => {
+        window.matchMedia = createMockMediaMatcher(true)
+      })
+
+      const element = (
+        <Media queries={{ sm: { maxWidth: 1000 }}}>
+          {({ sm }) => sm && <div>hello</div>}
+        </Media>
+      )
+
+      ReactDOM.render(element, node, () => {
+        expect(node.firstChild.innerHTML || '').toMatch(/hello/)
+      })
+    })
+  })
+
+  describe('with a render prop', () => {
     beforeEach(() => {
       window.matchMedia = createMockMediaMatcher(true)
     })
 
-    describe('and a children element', () => {
-      it('renders its child', () => {
-        const element = (
-          <Media query="">
-            <div>hello</div>
-          </Media>
-        )
-
-        ReactDOM.render(element, node, () => {
-          expect(node.firstChild.innerHTML).toMatch(/hello/)
-        })
+    it('should render expected result of the render prop call', () => {
+      beforeEach(() => {
+        window.matchMedia = createMockMediaMatcher(true)
+      })
+
+      const element = (
+        <Media
+          queries={{ sm: { maxWidth: 1000 }}}
+          render={({ sm }) => sm && <div>goodbye</div>}
+        />
+      )
+
+      ReactDOM.render(element, node, () => {
+        expect(node.firstChild.innerHTML || '').toMatch(/goodbye/)
       })
     })
+  })
+
+  describe('with a query that matches', () => {
+    beforeEach(() => {
+      window.matchMedia = createMockMediaMatcher(true)
+    })
+
+    const queries = {
+      sm: '(max-width: 1000px)',
+      lg: '(max-width: 2000px)',
+    }
+
+    const expected = {
+      sm: true,
+      lg: true,
+    }
 
     describe('and a children function', () => {
-      it('renders its child', () => {
-        const element = (
-          <Media query="">
-            {matches => (
-              matches ? <div>hello</div> : <div>goodbye</div>
-            )}
-          </Media>
-        )
-
-        ReactDOM.render(element, node, () => {
-          expect(node.firstChild.innerHTML).toMatch(/hello/)
-        })
+      it('should call children with expected values', () => {
+        ReactDOM.render(
+          <Media queries={queries}>
+            {matches => {
+              expect(matches).toMatchObject(expected)
```

Comment:
Let's test the output of render instead of the arguments to the render prop, just so we have complete coverage, and change the description to be something like "it renders its children function" or similar.